library/library/Book.cs

```diff
@@ -15,72 +15,42 @@ internal class Book
         public string Name { get; set; }
         public int GenreId { get; set; }
         public string Publisher { get; set; }
+        public string Author { get; set; }
+        public int Quantity { get; set; }
+        public DateTime ReleaseDate { get; set; }
 
 
-        public Book(string name, int genreId, string publisher)
+        public Book(string name, int genreId, string publisher, string author, int quantity, DateTime releaseDate)
         {
             Name = name;
             GenreId = genreId;
             Publisher = publisher;
+            Author = author;
+            Quantity = quantity;
+            ReleaseDate = releaseDate;
         }
-
         public void AddBook()
-            // Add book to DB via adapter
         {
             var connection = new SqlConnection(DbCon.ConnectionString);
             try
             {
                 connection.Open();
-                var adapter = new SqlDataAdapter("SELECT * FROM Book", connection);
-                var builder = new SqlCommandBuilder(adapter);
-                var table = new DataTable();
-                adapter.Fill(table);
 
-                var newRow = table.NewRow();
-                newRow["name"] = Name;
-                newRow["id_genre"] = GenreId;
-                newRow["publisher"] = Publisher;
-                table.Rows.Add(newRow);
+                var adapter = new SqlDataAdapter("AddBook", connection);
```

Comment:
It will be a good idea to descibe here in docstring why we are adding quantity to the book cuz table doeasnt contain that value. for example "using procedure altering copy table with this quantity property"